src/bulkDataRequestCache.ts

```diff
@@ -188,6 +188,28 @@ export class BulkUpsertRequestCache extends BulkDataRequestCache {
   }
 }
 
+export class BulkImportRequestCache extends BulkDataRequestCache {
+  public static getDefaultOptions(): TTLConfig.Options {
+    return {
+      isGlobal: true,
+      isState: true,
+      filename: BulkImportRequestCache.getFileName(),
+      stateFolder: Global.SF_STATE_FOLDER,
+      ttl: Duration.days(7),
```

Comment:
bulk ingest/query job results are available for 7 days after being created:
https://developer.salesforce.com/docs/atlas.en-us.252.0.salesforce_app_limits_cheatsheet.meta/salesforce_app_limits_cheatsheet/salesforce_app_limits_platform_bulkapi.htm